tasks/client_keys.yml

```diff
@@ -9,25 +9,25 @@
     mode: 0400
 
 - name: Generate client key
-  command: "openssl req -nodes -newkey rsa:{{ openvpn_rsa_bits }} -keyout {{ item }}.key -out {{ item }}.csr -days {{ openvpn_openssl_days }} -{{ openvpn_openssl_digest }} -subj /CN=OpenVPN-Client-{{  inventory_hostname[:24]  }}-{{  item[:24]  }}/"
+  command: "openssl req -nodes -newkey rsa:{{ openvpn_rsa_bits }} -keyout {{ item.name }}.key -out {{ item.name }}.csr -days {{ openvpn_openssl_days }} -{{ openvpn_openssl_digest }} -subj /CN=OpenVPN-Client-{{  inventory_hostname[:24]  }}-{{  item.name[:24]  }}/"
```

Comment:
why `[:24]` ?

Reply:
because the common name can at most be 64 characters